pick.lic

```diff
@@ -112,8 +112,9 @@ class LockPicker
   end
 
   def buy_lockpick(type)
-    fput("buy #{type} lockpick")
-    fput("offer #{@lockpick_costs[type]}")
+    response = dothis("buy #{type} lockpick", /prepared to/)
```

Comment:
We've created `bput` (in `common.lic`) as a better version of `put`, but I haven't seen `dothis` before. At a glance, `bput` checks for different error cases (I assume the checks in `dothis` are for Gemstone...). I also prefer the error messaging of `bput`. Thoughts @rcuhljr or @kamarcum ?